firmware/Makefile

```diff
@@ -2,8 +2,9 @@ all: signet-fw.dfu
 
 BTUPLE:=$(shell echo $(shell ./config.guess) | sed -e 's/\([a-z_09A-Z]\)*-/\1-build_/')
 HTUPLE:=arm-none-eabi
+export PATH := $(PWD)/toolchain/$(HTUPLE)/bin:$(PATH)
```

Comment:
I like this approach but I need to verify that we can actually use the same toolchain for both devices. They one is a Cortex M4 and another is a cortex M7. I think I made some changes to the crosstool configuration and there is a chance they broke compatibility and I'll need to maintain two toolchains and/or add some compiler switches to the Makefiles.

Reply:
As you say, they have to use different toolchains, they have different CPUs with different ct-ng/.configs.

They are located at formware/toolchain and firmware-hc/toolchain. But we use the same ct-ng to generate these toolchains.

Reply:
Oh right. As I've mentioned I've not built the legacy Signet firmware in quite awhile. I  should get on that and hunt around for any possible problems stemming from these changes. It looks safe but I'll give it a double check.